routes/web.php

```diff
@@ -124,6 +143,9 @@
     Route::get('/{id}/placedetails',[PlaceController::class, 'index'])->name('placedetails');
     # Place Favorite
     Route::post('/favorite/{place_id}/store', [PlaceFavoriteController::class, 'store'])->name('place_favorite.store');
+    Route::delete('/favorite/{place_id}/destroy', [PlaceFavoriteController::class, 'destroy'])->name('place_favorite.destroy');    
```

Comment:
for this I suggest to use `{place}` instead of `{place_id}` so that you can just directly inject the `Place` model in your controller

Reply:
in your `PlaceFavoriteController` destroy function, you can just say `destory(Place $place)` and inside it there's no need to say `::find($id)`

Reply:
These 2 are from other PR so we will fix them in another PR.